otelcolbuilder/Makefile

```diff
@@ -2,13 +2,31 @@ BINARY_NAME ?= otelcol-sumo
 BUILDER_VERSION ?= 0.62.0
 FIPS_SUFFIX ?= ""
 BUILDER_REPO ?= github.com/open-telemetry/opentelemetry-collector
-BUILDER_BIN_NAME ?= opentelemetry-collector-builder
-BUILDER_BIN_PATH ?= $(HOME)/bin
-INSTALLED_BUILDER_VERSION := $(shell opentelemetry-collector-builder version 2>&1)
 GO ?= go
+
+ifeq ($(OS),Windows_NT)
+	OS=windows
+endif
+
+ifeq ($(OS),windows)
+	BUILDER_BIN_EXT ?= .exe
+	MAKE := "$(shell cygpath '$(MAKE)')"
+endif
+
 OS ?= $(shell uname -s | tr A-Z a-z)
 ARCH ?= $(shell uname -m | sed s/aarch64/arm64/ | sed s/x86_64/amd64/)
 
+BUILDER_BIN_NAME ?= opentelemetry-collector-builder$(BUILDER_BIN_EXT)
+BUILDER_BIN_PATH ?= $(HOME)/bin
+
+ifeq ($(OS),windows)
+	BUILDER_BIN_UNIX_PATH ?= $(shell cygpath -u $(BUILDER_BIN_PATH))
```

Comment:
Is this env var `BUILDER_BIN_UNIX_PATH` used for anything? I cannot see any references to it.

Reply:
It was being used by curl previously but it seems to be working ok without it. I'll remove it.